cmd/tetragon/main.go

```diff
@@ -489,10 +489,15 @@ func Serve(ctx context.Context, listenAddr string, server *server.Server) error
 			log.WithError(err).Error("Failed to close gRPC server")
 		}
 	}(proto, addr)
-	go func() {
+	go func(proto, addr string) {
```

Comment:
nit: I believe that we don't need the arguments here. `proto` and `addr` are accessible inside this function.

Reply:
Indeed you are right! I guess I copied the function on top, used the same signature which is easy to read. We can keep it!